server/routes/v1/comment.ts

```diff
@@ -0,0 +1,82 @@
+import express, { Request, Response } from 'express';
+import db from '../../models';
+import CommentController from '../../controllers/v1/comment';
+import { getAuthUser } from '../../middleware/auth';
+
+const commentRouter = express.Router();
+const commentController = new CommentController(db.Post);
+
+commentRouter.get('', async (req: Request, res: Response) => {
+  try {
+    const limit = req.body.limit;
```

Comment:
These should be `req.query` for a GET request, we cannot send a body.

Reply:
makes sense!